frontend/app/playwright.config.ts

```diff
@@ -16,23 +16,21 @@ export default defineConfig({
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
-  retries: process.env.CI ? 1 : 0,
+  retries: 0,
```

Comment:
Are you sure about that? We should do that only if we are sure to not have flakiness in our current tests
Otherwise I'm afraid it's going to generate a lot of noise.

Reply:
In a nutshell, yes retries are not that useful in practices.
1. When a test fails, the 2nd attempt will fail anyways, adding 6min per failing test for nothing
2. Many times, there are false positives flakies: a test passes the 2nd time whereas it's should not (due to mutation during the 1st attempt etc...)
3. True flaky tests are never fixed, because the 2nd attempt will make it pass anyways

Reply:
LGTM then